index.html

```diff
@@ -136,7 +136,7 @@ <h3>Multi-Post Stories</h3>
                 </div>
             </section>
 
-            <section class="about">
+            <section class="flex col-direction about">
                 <h1>About<br> Myself</h1>
```

Comment:
- [ ] As suggested by the previous reviewer. Kindly make sure to use only one h1 tag in the entire document.